pom.xml

```diff
@@ -41,10 +41,13 @@
     <codahale.metrics.version>3.2.6</codahale.metrics.version>
     <javaxservlet.version>3.1.0</javaxservlet.version>
     <!-- Apache Ratis version -->
-    <ratis.version>2.2.0</ratis.version>
+    <ratis.version>2.3.0</ratis.version>
+    <!-- gRPC version, used for protoc-gen-grpc-java plugin to download -->
+    <!-- protoc-gen-grpc-java JAR -->
+    <grpc.version>1.44.0</grpc.version>
     <!-- ProtocolBuffer version, used to verify the protoc version and -->
     <!-- define the protobuf JAR version                               -->
-    <protobuf.version>3.5.1</protobuf.version><!-- Maven protoc compiler -->
+    <protobuf.version>3.19.2</protobuf.version><!-- Maven protoc compiler -->
```

Comment:
`grpc.version` and `protobuf.version` are aligned w/ `ratis.version`
https://github.com/apache/ratis/blob/ratis-2.3.0/pom.xml